pages/common/dd.md

```diff
@@ -19,6 +19,14 @@
 
 `dd if=/dev/zero of={{file_1GB}} bs=1024 count=1000000`
 
+- Generate a system backup into a img file and show the progress:
```

Comment:
```suggestion
- Create a system backup into a img file and show the progress:
```
thoughts?

Reply:
`Create` reads better than `Generate` IMO

Reply:
Is this an `img` file? I can restore any .img file into a partition?

Reply:
```suggestion
- Generate a system backup in the form of an img file and show the progress:
```
```suggestion
- Create a system backup in the form of an img file and show the progress:
```